pkg/database/migrations.go

```diff
@@ -1929,6 +1929,17 @@ func (db *Database) Migrations(ctx context.Context) []*gormigrate.Migration {
 					`ALTER TABLE authorized_app_stats DROP COLUMN IF EXISTS tokens_invalid`)
 			},
 		},
+		{
+			ID: "00083-ExpandTwilioFrom",
+			Migrate: func(tx *gorm.DB) error {
+				return multiExec(tx,
+					`ALTER TABLE sms_configs ALTER COLUMN twilio_from_number TYPE varchar(255)`)
+			},
+			Rollback: func(tx *gorm.DB) error {
+				return multiExec(tx,
+					`ALTER TABLE sms_configs ALTER COLUMN twilio_from_number TYPE varchar(16)`)
```

Comment:
Let's not rollback the size change, that _will_ lose data unnecessarily.

Reply:
sure - we do have similar rollbacks when we've done this in the past:
eg: `"ALTER TABLE verification_codes ALTER COLUMN long_code TYPE varchar(20)",`

Reply:
hmm yea we shouldn't roll back to a smaller column